posthog/models/personal_api_key.py

```diff
@@ -46,3 +50,68 @@ class PersonalAPIKey(models.Model):
         null=True,
         blank=True,
     )
+
+    @property
+    def scopes_list(self) -> list[str]:
+        return self.scopes.split(",") if self.scopes else []
+
+    @property
+    def scoped_organizations_list(self) -> list[str]:
+        return self.scoped_organizations.split(",") if self.scoped_organizations else []
+
+    @property
+    def scoped_teams_list(self) -> list[int]:
+        return [int(x) for x in self.scoped_teams.split(",")] if self.scoped_teams else []
+
+
+## API Scopes
+# These are the scopes that are used to define the permissions of the API tokens.
+# Not every model needs a scope - it should more be for top-level things
+# Typically each object should have `read` and `write` scopes, but some objects may have more specific scopes
+
+# WARNING: Make sure to keep in sync with the frontend!
```

Comment:
That makes me wonder... could we leverage the schema export? Like, take all fields off of the resulting Pydantic model as the list of scopes.

Reply:
We definitely _could_ - just felt like a fair amount of effort for now